extensions/vscode/src/utils/errors.ts

```diff
@@ -12,6 +12,15 @@ export const getStatusFromError = (error: unknown): number | undefined => {
   return undefined;
 };
 
+export const getStatusStringFromErrorResponse = (
+  error: unknown,
+): string | undefined => {
+  if (axios.isAxiosError(error)) {
+    return error.response?.statusText;
+  }
+  return undefined;
+};
+
```

Comment:
The present of this `statusText` value proved to be helpful (when we received some of the more obscure HTTP error codes in a response).